packages/perseus/src/widgets/interactive-graphs/graphs/components/movable-point.tsx

```diff
@@ -14,6 +13,10 @@ type Props = {
     color?: string;
 };
 
+// The hitbox size of 48px by 48px is preserved from the legacy interactive
+// graph.
```

Comment:
Not only legacy. I remember Sarah B mentioning this is also the Android minimum tap target size. 

https://support.google.com/accessibility/android/answer/7101858

But the fun begins, the current [Material Design](https://m3.material.io/foundations/accessible-design/accessibility-basics#20bcdd28-186d-4884-b592-3fb069e10a73) from Google mentions 44 (which matches [WCAG](https://www.w3.org/WAI/WCAG21/Understanding/target-size.html#:~:text=If%20the%20target%20is%20the,44%20by%2044%20CSS%20pixels.&text=A%20footnote%20or%20an%20icon,from%20the%20minimum%20target%20size.)).

🤷‍♂️

Reply:
Actually, I read wrong.  The Material Design spec makes a distinction between "Touch and pointer" (48 dp) and strictly "pointer" targets (44 dp).

Ignore me. :)